data/hai/hai reveal 4 middle.txt

```diff
@@ -1413,13 +1413,133 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	She spreads her arms, 'palms' up, in a human gesture indicating a loss for words. "Our system of government has systems and processes to allow almost any internal situation to be handled without resorting to base hysteria, and to actively punish and isolate those who try and subvert those processes. It is a difficult pill to swallow sometimes, but it works, and it has ensured stability since before humans learned to bash rocks together. Aimet is the master navigator of those systems. He's responsible for improving one of them for the first time in two thousand years."`
+			
+			label return
+			`	Teeneep's console beeps a moment prior to Sayari's slightly breathless return. A message, entirely in your language, is displayed on the screen; it announces an emergency meeting of the full Elder Council in under an hour.`
+			`	Sayari steps across to Teeneep and glances at the message. "I needn't have left this room. We must prepare quickly, we're going to have an escort..."`
+			`	Teeneep's expression darkens, but she quickly ensures she has what she needs before a group of Council security staff arrive to take you to the chambers. This is not the first time you've seen the room, but it is the first time you've seen it full. There is a small desk and seat at the front of the central circle, opposite the entry point, which was not occupied when you entered here for dealing with Scar's Legion but now hosts a very serious Hai bearing unusual marks of office. Sayari sees your line of vision and nudges you, conveying in a whisper, "That's the moderator. They're only required when more than two thirds of the council is in attendance, and when there's a debate or petition to address."`
+			`	She glances around the room and raises an eyebrow. "This is a closed session, which is quite unusual for a full Council Assembly. This does not bode well."`
+			`	As the security staff take their place, the moderator walks over to address you while the Elders are still filtering in and muttering to each other. "Now, before we begin, I would like to confirm. I understand you are able to act as a witness, and confirm that the threat carried from the Unfettered was indeed real. Is this correct?"`
+			choice
+				`	"That is correct."`
+					goto correct
+				`	"I can also testify to the human position on the matter."`
+			`	The moderator pulls a strange face and says, "Right, well, I doubt that will be necessary.`
+			label correct
+			`	"Thank you, Captain. Since you have become involved as a witness, protocol dictates that the session will be held in your language, though for the sake of order normal council directions will still be in Hai. We will begin in a moment."`
```

Comment:
It's order because Hai who are less versed in your language will respond more quickly to moderator declarations in Hai.

However, it occupies the same grammatical point so it either does or does not either way. Following on from a "though" I don't believe it should have one.